### PR TITLE
[COD-34] Implement refresh tokens endpoint and connect it to frontend v.2

### PR DESCRIPTION
**Pull request by SilaEwGenii**
Hello. 
https://legolas.atlassian.net/browse/COD-34
Implemented "refresh tokens" endpoint and connect it to frontend and add testing for service

### CODE REVIEW

**Review by npupko — APPROVED** [review]
LGTM

File: backend/spec/services/sessions/refresher_spec.rb [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
+  subject(:refresh) { described_class.new.call(token_params:) }
 
   let(:user) { create(:user) }
   let(:issuer) { Jwt::Issuer.call(user) }
 
   let(:token_params) do
     {
       token: issuer[0],
-      refresh_token: issuer[1].token
+      refreshToken: issuer[1].token
```
Comment (Set27) [inline backend/spec/services/sessions/refresher_spec.rb:14]: refresh_token???

File: backend/spec/requests/api/v1/sessions_spec.rb [hunk 2]
```diff
@@ -45,14 +45,46 @@
...
+
+      it 'returns 201 OK' do
+        post('/api/v1/refresh', params:)
+        expect(response).to have_http_status(:created)
+      end
+
+      it 'returns new token pair' do
+        post('/api/v1/refresh', params:)
+        expect(JSON.parse(response.body)).to include({ 'token' => a_kind_of(String),
+                                                       'refresh_token' => a_kind_of(String) })
```
Comment (Set27) [inline backend/spec/requests/api/v1/sessions_spec.rb:67]: а это чё?

Comment (Set27) [reply backend/spec/requests/api/v1/sessions_spec.rb:67]: 'refresh_token'

**Review by Set27 — APPROVED** [review]

**Pull request merged**